utils/has-negation-inside-parens.ts

```
@@ -20,7 +19,7 @@ export function hasNegationInsideParens(
   node: Node,
   context: Rule.RuleContext,
 ): boolean {
-  let sourceCode = getNodeContent(node, context)
+  let sourceCode = context.sourceCode.getText(node)
```

ESLint

[error] Unsafe argument of type error typed assigned to a parameter of type `Node | undefined`.

(typescript/no-unsafe-argument)